home.html

```diff
@@ -0,0 +1,141 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Shop-Ship/Home</title>
+    <link rel="stylesheet" href="https://garuda-ui.netlify.app/index.css">
+    <link rel="stylesheet" href="/home.css">
+    <script src="https://kit.fontawesome.com/108bfd668b.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <header class="e-com-header">
+        <nav class="e-com-navbar">
+            ShopShip
+            <ul class="e-com-nav-items">
+            </ul>
+            <input type="text" class="search-input" placeholder="search">
+            <div class="e-com-social">
+                <div class="ecom-login-container">
```

Comment:
In code, your class name is e-com-... , but here you write ecom. Why???

Reply:
i will correct it